VocaDbWeb/Controllers/TagController.cs

```diff
@@ -237,7 +237,6 @@ public ActionResult Merge(int id, int? targetTagId)
 			return RedirectToAction("Edit", new { id = targetTagId.Value });
 		}
 
-		[ResponseCache(Location = ResponseCacheLocation.Any, Duration = 3600)]

```

Comment:
Can we simply disable ResponseCache, or should we change it to `[ResponseCache(Location = ResponseCacheLocation.Any, Duration = 3600, VaryByQueryKeys = new[] { "id", "lang" })]`?